1.purpose_and_goals.md

```diff
@@ -2,9 +2,9 @@
 
 The Open Application Model defines a standard, platform-agnostic way to describe cloud and edge applications. The goal of the specification is to provide a common way to describe applications agnostic to any specific container runtime, orchestration software, cloud provider, or hardware configuration, with clearly defined roles for developers and operators, while still allowing implementations to make use of the native APIs, tools, and features that are unique to the implementation and its underlying platform.  
 
-The central problem this specification seeks to address is how distributed applications can be composed and handed off to those responsible for operating them. The problem is not so much about how to write programs, but how to take components of a service-oriented (or microservice-oriented) architecture, and streamline the workflow that surrounds such applications.
+The central problem this specification seeks to address is how distributed applications can be composed and handed off to those responsibilities for operating them. The problem is not so much about how to write programs, but how to take components of a service-oriented (or microservice-oriented) architecture, and streamline the workflow that surrounds such applications.
```

Comment:
we need a `verb` like responsible here, not a `noun`

Reply:
Yes, I see , it means that 

*The central problem this specification seeks to address is how distributed applications can be composed and handed off to those **(that are)** responsible for operating them.*, this should be `responsible`.